Source/Core/VideoCommon/FrameDump.cpp

```diff
@@ -447,14 +444,15 @@ void FrameDump::CheckForConfigChange(const FrameData& frame)
   {
     Stop();
     ++m_file_index;
-    PrepareEncoding(frame.width, frame.height);
+    PrepareEncoding(frame.width, frame.height, frame.state.ticks, frame.state.savestate_index);
```

Comment:
I'm using `frame.state.ticks` here to match master's behavior, but master's behavior seems fishy to me. Doesn't this discard the information about how much time passed between the first frame of this file and the last frame of the previous file? And if so, would the correct thing to use here be the time of the previous frame, or the time of the previous frame plus one "timebase unit"...?

Reply:
After looking into the ffmpeg API some more, I think the right thing to do would be to explicitly set the duration of the old stream and to use `frame.state.ticks` here. I will leave setting the duration for another PR.

Reply:
After looking into this some more, I'm not sure if there's any solution that works great... other than implementing writing audio data to the AVI file, which is a desirable feature regardless of this problem. I think we should move forward with this PR as it is.